cep-999.md

```diff
@@ -0,0 +1,162 @@
+This path is "Lib/site-packages" if the environment targets a Windows platform and "lib/pythonX.Y/site-packages" on other platforms where X and Y and the major and minor versions of the `python` package installed in the environment.
+This matches the current behavior of conda, mamba and pixi.
+
+This field should only be included in the metadata for `python` packages but tools must not fail if packages with other names include this field, rather the entry should be ignored.
+
+To avoid unnecessarily increasing the size of repodata.json, it is highly recommended that this field only be included in `python` packages where it is required, that is where the site-packages directory is not the default value.
+Packages with other names or `python` packages that use the default site-packages path should not include this field.
+
+This path must not point to a location outside of the root of the environment (e.g. it cannot navigate up a directory or be an absolute path).
+If a package specifies a path which violates this requirement the package must not be installed and an appropriate error should be shown to the user.
+
+A possible way to check the validity of the `python_site_packages_path` field is with this function:
+
+``` Python
+def is_valid(target_prefix: str, python_site_packages_path: str) -> bool:
+    target_prefix = os.path.realpath(target_prefix)
+    full_path = os.path.realpath(os.path.join(target_prefix, python_site_packages_path))
+    test_prefix = os.path.commonpath((target_prefix, full_path))
+    return test_prefix == target_prefix
+```
+
+When a package which includes this optional field is indexed the `python_site_packages_path` field will be included in the repodata entry for the package.
+For example the repodata entry for a python-3.13.0rc1 package using the free-threading build configuration might look as follows:
+
+```
+    "python-3.13.0rc1-haa6bb3f_0_cpython_cp313t.tar.bz2": {
+      "build": "haa6bb3f_0_cpython_cp313t",
+      "build_number": 0,
+      "depends": [
+        "bzip2 >=1.0.8,<2.0a0",
+        "expat >=2.6.2,<3.0a0",
+        "libffi >=3.4,<4.0a0",
+        "ncurses >=6.4,<7.0a0",
+        "openssl >=3.0.14,<4.0a0",
+        "readline >=8.1.2,<9.0a0",
+        "sqlite >=3.45.3,<4.0a0",
+        "tk >=8.6.14,<8.7.0a0",
+        "tzdata",
+        "xz >=5.4.6,<6.0a0",
+        "zlib >=1.2.13,<1.3.0a0"
+      ],
+      "license": "PSF-2.0",
+      "license_family": "PSF",
+      "timestamp": 1722610680768,
+      "track_features": "free-threading",
+      "md5": "c09289eb86239e1221533457d861f1a3",
+      "name": "python",
+      "size": 16332720,
+      "subdir": osx-arm64,
+      "version": "3.13.0rc1",
+      "sha256": "fa0ae22c13450fe6c30c754ee5efbd7fe7e7533b878d7be96e74de56211d19df",
+      "python_site_packages_path": "lib/python3.13t/site-packages"
+    },
+```
+
+This field will also be included in repodata derivatives, like current_repodata.json or sharded repodata when appropriate.
```

Comment:
`current_repodata.json` is less important ever since the libmamba solver has been rolled out, do you think it would be a dealbreaker if we'd not support this in that file or am I jumping the gun on this one?

Reply:
It is not a dealbreaker to not include the field in `current_repodata.json`.  The field should be included in a `python` shard of repodata if/when that becomes a reality. I don't see including this as a big lift in either case as the field is tied to the package entry from which I expect both to be derived.

Reply:
Individual entries in current_repodata and sharded repodata will be the same as the ones in monolithic repodata.